package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,21 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = [line[6:15], line[15:24],
```

Comment:
My bad here for not realising this sooner, but the way this construct works now, you won't be triggering the `ValueError` if a non-numerical CRYST1 record is read in, i.e. if `line[6:15] == "A"` this would be perfectly normal assignement for `cell_dims`.

This however is an issue as we would expect an error to be thrown (hence the test failures).

To fix this I would suggest doing:

```python
cell_dims = np.array([line[6:15], line[15:24], line[24:33], line[33:40],
                     line[40:47], line[47:54]], dtype=np.float32)
```

With the associated changes for llines 402-404, this _should_ work.